connectivity/FEATURE_BLE/include/ble/Gap.h

```diff
@@ -409,8 +412,9 @@ class Gap {
         }
 
         /**
-         * Called when connection attempt ends or an advertising device has been
-         * connected.
+         * Called when connection attempt ends. Check event.getStatus() to see if connection
+         * was established. If this device is the peripheral and it as advertising this will
+         * end advertising which will also create the onAdvertisingEnd event.
```

Comment:
```suggestion
         * Called when connection attempt ends. Check event.getStatus() to see if connection
         * was established. If this device is the peripheral and it was advertising this will
         * end the advertising set which will also create the onAdvertisingEnd event.
```